core/src/main/scala/io/delta/tables/DeltaTableBuilder.scala

```diff
@@ -16,23 +16,20 @@
 
 package io.delta.tables
 
-import scala.collection.mutable
```

Comment:
cleaned up some unused imports below

Reply:
Could you revert the import changes? The IntelliJ's default import order doesn't match our style.

Reply:
reverted mostly, there were still some unused imports, should i keep those too?

Reply:
This looks good to me